### PR TITLE
Orientation Login bugs

### PR DESCRIPTION
**Pull request by koziodigitalinc**

### CODE REVIEW

**Comment by smarx** [comment]
*Automated message from Dropbox CLA bot*

@koziodigitalinc, thanks for the pull request! It looks like you haven't yet signed the Dropbox CLA. Please [sign it here](https://opensource.dropbox.com/cla/).

File: app/src/main/java/com/zulip/android/activities/LoginActivity.java [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/activities/LoginActivity.java]: Can you remove this line?

File: app/src/main/java/com/zulip/android/activities/LoginActivity.java [hunk 2]
```diff
@@ -52,6 +54,12 @@
     private EditText mUserName;
     private EditText mPassword;
     private EditText serverIn;
+    //region state-restoratoin
```
Comment (niftynei) [inline app/src/main/java/com/zulip/android/activities/LoginActivity.java]: typo here, this should be state-restorat**io**n ;)

File: build.gradle [hunk 3]
```diff
@@ -8,6 +8,7 @@ buildscript {
         classpath 'com.google.gms:google-services:2.1.0'
     }
 }
+project.ext.fabricKey = "";
```
Comment (niftynei) [inline build.gradle]: Can you explain what this line is for?  The `fabricKey` property is only pertinent to the `release` builds of the Zulip android app, not all the projects.

**Comment by niftynei** [comment]
Hey @koziodigitalinc thanks for the commits. Small note that for commit messages, we ask that they all be capitalized, to match the other commits in the project.

File: build.gradle (hunk 3 above)
Comment (koziodigitalinc) [inline build.gradle]: Basically the build freaks out if there's no Fabric key. Not sure if you get this but it throws an error on a gradle sync stating that `fabricKey` doesn't exist

Comment (niftynei) [inline build.gradle]: I see. Your build is attempting to build the release candidate as well as the debug version, which is why you're running into this error.

You're right that it needs to be fixed but I'd prefer a less global way of doing it.  

Can you put a fix for this into a separate pull request?  It's tangential to the Orientation Login bugs.

**Comment by niftynei** [comment]
Hey @koziodigitalinc can you condense your changes down into a single, logical commit?  You can read more about the Zulip version control guidelines here: http://zulip.readthedocs.io/en/latest/version-control.html

File: build.gradle (hunk 3 above)
Comment (koziodigitalinc) [inline build.gradle]: Sure can do!

File: app/src/main/java/com/zulip/android/activities/LoginActivity.java (hunk 1 above)
Comment (koziodigitalinc) [inline app/src/main/java/com/zulip/android/activities/LoginActivity.java]: Done. only one line between the two now.

**Comment by koziodigitalinc** [comment]
@niftynei Alright, i squashed down the commits. I think we're pretty much good (did updates too)

**Comment by niftynei** [comment]
Thanks!
